drivers/gpio/gpio_mcux.c

```diff
@@ -17,6 +17,16 @@
 
 #include <zephyr/drivers/gpio/gpio_utils.h>
 
+#if defined(CONFIG_SOC_SERIES_IMXRT7XX) && defined(CONFIG_SOC_FAMILY_NXP_IMXRT)
```

Comment:
Please improve the commit message. Update GPIO driver doesn't really tell us much. Please offer a short description of what is the GPIO driver model on imxrt7xx.